test/e2e/test_translate_proxy.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I know this is annoying, but better get ahead of the curve... Can you remove the year from the copyright notice? I only bring this up because it's a new file, at some point we'll have to do it to all files.

Reply:
Removed date.